sdk/reservations/Microsoft.Azure.Management.Reservations/src/Microsoft.Azure.Management.Reservations.csproj

```diff
@@ -7,11 +7,14 @@
 		<PackageId>Microsoft.Azure.Management.Reservations</PackageId>
 		<Description>Microsoft.Azure.Management.Reservations Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Reservations</AssemblyName>
-		<Version>1.11.0-preview</Version>
+		<Version>1.12.0-preview</Version>
     <PackageTags>Reservations;</PackageTags>
 		<PackageReleaseNotes>
 			Enhancing Patch API for RI renewal.
 		</PackageReleaseNotes>
+		<PackageReleaseNotes>
```

Comment:
You should update the content of above `<PackageReleaseNotes>` directly instead of adding a new `<PackageReleaseNotes>`